jsoniter-scala-benchmark/shared/src/main/scala/com/github/plokhotnyuk/jsoniter_scala/benchmark/ADTReading.scala

```diff
@@ -49,7 +50,7 @@ class ADTReading extends ADTBenchmark {
   def jsoniterScala(): ADTBase = readFromArray[ADTBase](jsonBytes)
 
   @Benchmark
-  def ninnyJson(): ADTBase = ninny.Json.parse(new String(jsonBytes, UTF_8)).to[ADTBase].get
+  def ninnyJson(): ADTBase = ninny.Json.parseArray(ArraySeq.unsafeWrapArray(jsonBytes)).to[ADTBase].get
```

Comment:
Is the proposed version with an array of boxed bytes faster than creation of string and parsing from it?

Reply:
The actual parsing probably isn't any faster. Theoretically it won't box and will benchmark better due to using less memory since it's only wrapping an array in a class rather than copying the data into `String`. Practically I wasn't able to see a difference in this benchmark.